sdk/docs/manually-written/sdk/component-howtos/smart-contracts/flags.rst

```diff
@@ -1,112 +1,146 @@
-- | ``--enable-multi-package ARG``
-  | Enable/disable multi-package.yaml support
-  | Can be set to "yes", "no" or "auto" to select the default (yes)
-- | ``--all``
-  | Build all packages in multi-package.yaml
-- | ``--no-cache``
-  | Disables cache checking, rebuilding all dependencies
-- | ``--multi-package-path FILE``
-  | Path to the multi-package.yaml file
+These flags are available for both ``--build`` as well as ``--test``.
 
-Includes :ref:`General Daml Compiler flags <general-damlc-flags>`
+- | ``-h``, ``--help``
 
-``clean``
-==============
+  | Prints option information when called on the commandline.
 
-- | ``--all``
-  | Clean all packages in multi-package.yaml
-- | ``--enable-multi-package ARG``
-  | Enable/disable multi-package.yaml support
-  | Can be set to "yes", "no" or "auto" to select the default (yes)
-- | ``--multi-package-path FILE``
-  | Path to the multi-package.yaml file
+- | ``--package-root ARG``
 
-``test``
-=============
+  | :subsiteref:`DPM-specific<dpm>` option.
 
-- | ``--files FILE``
-  | Only run test declarations in the specified files.                     
-- | ``--all``
-  | Run tests in current project as well as dependencies
-- | ``--load-coverage-only``
-  | Don't run any tests - only load coverage results from files and write the aggregate to a single file.
-- | ``--show-coverage``
-  | Show detailed test coverage
-- | ``--color``
-  | Colored test results
-- | ``--junit FILENAME``
-  | Filename of JUnit output file
-- | ``-p,--test-pattern PATTERN``
-  | Only scripts with names containing the given pattern will be executed.
-- | ``--transactions-output ARG``
-  | Filename to which the transaction list should be output
-- | ``--load-coverage ARG``
-  | File to read prior coverage results from. Can be specified more than once.
-- | ``--save-coverage ARG``
-  | File to write final aggregated coverage results to.
-- | ``--coverage-ignore-choice ARG``
-  | Remove choices matching a regex from the coverage
-  | report. The full name of a local choice takes the
-  | format '<module>:<template name>:<choice name>',
-  | preceded by '<package id>:' for nonlocal packages.
+  | Path to the root of a :ref:`glos-daml-package` containing ``daml.yaml``.
+    Using this option, you set the package to build/test. You should prefer the
+    ``DAML_PACKAGE`` environment variable over this option. See
+    :subsiteref:`dpm-configuration` for more details.
+
+- | ``--project-root ARG``
+
+  | :subsiteref:`DPM-specific<dpm>` option.
+
+  | ``project-root`` is **deprecated**, please use ``--package-root``.
+
+  | Path to the root of a :ref:`glos-daml-package` containing ``daml.yaml``. You
+    should prefer the ``DAML_PACKAGE`` environment variable over this option.
+    See :subsiteref:`dpm-configuration` for more details.
+
+- | ``--package-check``
+
+  | Check if running in Daml :ref:`glos-daml-package`.
 
-Includes :ref:`General Daml Compiler flags <general-damlc-flags>`
+  | ?????
```

Comment:
It's good that you are calling it out. We need a better description of what this option is doing
@samuel-williams-da can you chip-in?